hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/common/model/HoodieSparkRecord.java

```diff
@@ -83,6 +83,7 @@ public class HoodieSparkRecord extends HoodieRecord<InternalRow> {
    */
   private final transient StructType schema;
 
+  private boolean isDeleted;
```

Comment:
can you add java docs as to how to deduce isDeleted

Reply:
Done